docs/user/00-10-istio-version.md

```diff
@@ -0,0 +1,29 @@
+# Istio Version
+Learn which version of Istio the Istio module contains and how to enable compatibility mode with the previous minor version of Istio.
+
+The version of Istio depends on the version of the Istio module that you use. If a new version of the Istio module introduces a new version of Istio, an upgrade of the module causes an automatic upgrade of Istio.
+
+The latest release includes the following versions of Istio and Envoy:  
+
+**Istio version:** 1.22.3
+
+**Envoy version:** 1.30.5
```

Comment:
It is, but if it's not a problem, I'd rather update it with another PR. Developers take care of updating the information about the versions and now I need to let them know about the change and update our guidelines, so this information is mentioned in the open-source release notes instead of here